North-Seattle-College/ad440-winter2022-thursday-repo
# Frontend UI Maintenance

# Description
Closes #57 
This is a major fronted maintenance and quality control PR for the Single Page Application _(SPA)_ which focuses on the following items:
- Condensing all `CSS` into a single file.
- Providing a basic single unifying look to the app's UI
- Removing unneeded and unused files.
- Formatting all the frontend code.
- Fixing all console warnings

## To Test
1. `git checkout feature/SPA-maintenance`
2. `cd frontend`
3. `npm install`
4. `npm start`
5. Test all the functions of the page _(Text entry, submit, and clear)_
6. Verify all features work properly
7. Verify all elements of the page load and display correctly

## Ideas For Future Frontend Tasks/Improvements
- [ ] Create a more intuitive UI that matches Floops mockups and company design
- [ ] Headers are aligned to the start of the boxes below them.
- [ ] Submit and Clear are aligned to the end of the input box.
- [ ] Add a credits section that can link to this project give background on the application.
- [ ] Pressing the enter key when tying a message should behave the same way as clicking "Submit".
- [ ] Have the AI Feedback section always be visible to the user (At least when a message is being edited).
- [ ] Create catches for invalid entries such as empty strings.
- [ ] Create an error response to display if an invalid entry is entered or an API returns an error. 

## Time
- Estimated: _1-3 hours_
- Actual: _4.5 hours_
